src/Licenses/LicenseFindFilterInterface.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses;
+
+use ArrowSphere\PublicApiClient\Licenses\Entities\License\ActiveSeats;
+use ArrowSphere\PublicApiClient\Licenses\Entities\License\License;
+use ArrowSphere\PublicApiClient\Licenses\Entities\License\Price;
+use ArrowSphere\PublicApiClient\Licenses\Entities\LicenseOfferFindResult;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\ActionFlags;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\Offer;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\ActionFlags as PriceBandActionFlags;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\Billing;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\PriceBand;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\Prices;
+use ArrowSphere\PublicApiClient\Licenses\Entities\Offer\PriceBand\SaleConstraints;
+
+/**
+ * Interface LicenseFindFilterInterface
+ *
+ * Description
```

Comment:
Replace "Description" with this text:
These consts are meant to be used with LicensesClient, with DATA_KEYWORDS, DATA_FILTERS and DATA_SORT options.